python-checks/src/main/java/org/sonar/python/checks/hotspots/GraphQLIntrospectionCheck.java

```diff
@@ -139,6 +149,13 @@ private static Optional<String> nameFromIdentifierOrCallExpression(Expression ex
     return Optional.ofNullable(TreeUtils.nameFromExpression(expression))
       .or(() -> TreeUtils.toOptionalInstanceOf(CallExpression.class, expression)
         .map(CallExpression::callee)
-        .map(TreeUtils::nameFromExpression));
+        .flatMap(GraphQLIntrospectionCheck::nameFromExpressionOrQualifiedExpression));
+  }
+
+  private static Optional<String> nameFromExpressionOrQualifiedExpression(Expression expression) {
+    return Optional.of(expression)
+      .flatMap(TreeUtils.toOptionalInstanceOfMapper(QualifiedExpression.class))
```

Comment:
Here you could use `TreeUtils.toOptionalInstanceOf()`

Reply:
Done in c3c9a320bf6afa8ea03d9bc9a8446be257743ef4.